tyk-headless/templates/deployment-gw-repset.yaml

```
@@ -122,15 +122,65 @@ spec:
                 key: APISecret
           - name: TYK_GW_HTTPSERVEROPTIONS_USESSL
             value: "{{ .Values.gateway.tls }}"
-            {{- if .Values.pump.enabled }}
           - name: TYK_GW_ENABLEANALYTICS
+            value: "{{ .Values.pump.enabled }}"
+          # Default Settings
+          - name: TYK_GW_TEMPLATEPATH
+            value: "/opt/tyk-gateway/templates"
+          - name: TYK_GW_TYKJSPATH
+            value: "/opt/tyk-gateway/js/tyk.js"
+          - name: TYK_GW_MIDDLEWAREPATH
+            value: "/mnt/tyk-gateway/middleware"
+          - name: TYK_GW_USEDBAPPCONFIGS
+            value: "false"
+          - name: TYK_GW_APPPATH
+            value: "/mnt/tyk-gateway/apps"
+          - name: TYK_GW_ANALYTICSCONFIG_TYPE
+            value: "mongo"
```

Should it be empty or removed? I believe `tyk-headless` does not communicate with mongo? 

```suggestion
            value: ""
```